content/post/benchmarking-gnns/index.md

```diff
@@ -0,0 +1,268 @@
+
+Now, if we extract results of both these papers to put together in the same table and claim that the model with the highest performance score is the promising of all, **can we get convinced** that the comparison is fair?
+
+>There are other issues related to hyperparamter selection, comparison in an unfair budgets of trainable paramters, use of different train-validation-test splits, etc.
+
+The existence of such problems pushed us to develop a GNN benchmarking framework which **standardizes GNN research** and help researchers make more meaningful advances.
+
+---
+
+### Challenges of building a GNN benchmark
+
+The lack of benchmarks have been a major issue in GNN literature as the **aforementioned requirements have not been strictly enforced**.
+
+We believe that a standard and unified benchmark framework should have --
+
+1. an easy to use and reproducible coding framework
+2. rigorous and fair experimental setting
+3. appropriate datasets that can statistically separate model performance
+4. comprehensive in terms of the fundamental tasks (applications) that the research can be applied to. 
+
+> Having said that, designing benchmarks is highly challenging as we must make robust decisions for the 4 key points noted above. 
+
+Particularly for the point 3, it is **challenging to collect real and representative large-scale datasets**. The lack of theoretical tools that can define the quality of a dataset _or,_ validate its statistical representativeness for a given task makes it difficult to decide on datasets. Furthermore, there are arbitrary choices required on the features of nodes and edges for graphs and the scale of graph sizes as most of the popular graph learning frameworks do not cater _‘very efficiently’_ to large graphs. 
+
+> There has been a promising effort recently, [The Open Graph Benchmark (OGB)](https://ogb.stanford.edu), to collect meaningful medium-to-large scale dataset in order to steer graph learning research. The initiative is complementary to the goals of this project.
+
+
+---
+
+### Proposed benchmarking framework:
+We include each of the four characteristics listed out in the previous section to propose a benchmarking framework. 
+
+1. We develop a modular coding infrastructure which can be used to speed up the development of new ideas 
+2. Our framework adopts a rigorous and fair experimental protocol, 
+3. We propose appropriate medium-scale datasets that can be used a plug-ins for later research. [^2]  
+[^2]: As examples, you may refer to [these](https://github.com/lukecavabarrett/pna) [works](https://github.com/AITRICS/mol_reliable_gnn) that leverage our framework to conveniently work on their research idea. It indicates the effectiveness of having such a framework.
+4. Four fundamental tasks in graph machine learning are covered, i.e. graph classification, graph regression, node classification, and edge classification.
```

Comment:
I feel like this above 4 point list and the previous one are so similar that they may be combined some how? (A suggestion)